src/CommunityToolkit.Maui.MediaElement/CommunityToolkit.Maui.MediaElement.csproj

```diff
@@ -63,6 +65,9 @@
     <PackageReference Include="Xam.Plugins.Android.ExoPlayer.Transformer" Version="[2.19.1,)" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiPackageVersion)"/>
+    <PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiPackageVersion)" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Mopups" Version="1.3.0" />
```

Comment:
Why are we adding a reference to Mopups? 

Preferably, we should use `CommunityToolkit.Maui.Views.Popup`.

Reply:
I tried that first @brminnick. I had too many issues with sizing of elements and not fitting the window. I had multiple CTD issues that were resolved immediately by using mopup. If Community toolkit Popup is fixed for android and IOS it would be great idea. But I have spent weeks trying that. It took hours to get mopup working.

Reply:
What are the issues that you are seeing with Popup? I agree that we should avoid bringing in new dependencies if we can help it